### DISCUSSION TITLE
Shared eslint and stylelint configs

### DISCUSSION THREAD

**Discussion by dimitrov-adrian** [directus/directus#8511] [Feature Requests]
It will be nice to have shared eslint and stylelint configurations, so when creating extensions to benefit and follow the same style as Directus code.

Info:
- https://eslint.org/docs/developer-guide/shareable-configs
- https://stylelint.io/user-guide/get-started/#your-configuration

Examples:
- https://www.npmjs.com/package/eslint-config-airbnb
- https://www.npmjs.com/package/@shopify/stylelint-plugin

**Comment by rijkvanzanten** [comment]
@dimitrov-adrian Would you prefer two separate packages (eg `@directus/eslint-config` & `@directus/stylelint-config`, or would you prefer a single `@directus/linter` that exports both?

**Comment by dimitrov-adrian** [comment]
Don't have any preferences, as soon if it works. What I saw from other projects, they use as separate packages. So `@directus/eslint-config` and `@directus/stylelint-config` sounds in place.

**Reply by rijkvanzanten** [reply]
@azrikahar Can you help get this going? 🙂

**Reply by azrikahar** [reply]
Ok 👌

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
